### ISSUE TITLE
netlify-cms styles are leaking into the rest of my app

### ISSUE THREAD

**Issue by AJTJ** [decaporg/decap-cms#3992]
I'm using the `netlify-cms-app` in order to leverage the cms as a static route within my `react-static` app. I'm very much using this experience as a template => https://formidable.com/blog/2020/oss-cms/

I've just started the app and I'm using `emotion` (barely) to style it. When I go to the `netlify-cms` route and then go back to another route, the `netlify-cms` styles will now affect and even override my app's styles.

my repo is located here: https://github.com/AJTJ/ylc_website

I'm not even sure what information I should provide for this.

**Comment by bever1337** [comment]
Have you considered not routing back to the main app with an SPA/client-side router? I've encountered this bug too, and the solution was just to use a vanilla, HTML link that escapes client-side routing.

There's been lots of progress to make the CMS work better as a 'subapp' since I first adopted it. It is probably an absurd, buzzword'y lift that requires lots of testing, but I wonder if using a ShadowDOM to encapsulate styles would work here?
